fabric-object-builder-api-v1/src/main/java/net/fabricmc/fabric/api/object/builder/v1/ai/tasks/VillagerTaskProviderRegistry.java

```diff
@@ -0,0 +1,84 @@
+package net.fabricmc.fabric.api.object.builder.v1.ai.tasks;
+
+import com.google.common.collect.ImmutableMap;
+import com.mojang.datafixers.util.Pair;
+import net.minecraft.entity.ai.brain.task.Task;
+import net.minecraft.entity.passive.VillagerEntity;
+import net.minecraft.village.VillagerProfession;
+//import org.jetbrains.annotations.Contract;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.function.BiFunction;
+
+public final class VillagerTaskProviderRegistry {
+	private static final Map<VillagerProfession, VillagerTaskProvider> VILLAGER_TASK_PROVIDER_BUILDER = new HashMap<>();
+	
+	/**
+	 * Adds a {@link VillagerTaskProvider} to the registry, which will be called by the associated {@link VillagerProfession}.
+	 * @param executingProfession The {@code VillagerProfession} that will execute this {@code VillagerTaskProvider}.
+	 * @param taskListProvider The {@code VillagerTaskProvider} to register.
+	 */
+	public static void register(final VillagerProfession executingProfession, final VillagerTaskProvider taskListProvider)
+	{
```

Comment:
The brackets shouldn’t be like this, I think it might even fail a checkstyle check.

Reply:
I switched to that style for personal use, because I find it easier to read when the header is separate from the body.  I'll run my IDE's code cleanup with the default preset.

Reply:
Check the github actions failed logs to know what needs fixing. It doesn't include everything that needs fixing, but it won't stop failing until it fixes everything. You can also attempt to check locally by trying to build the module and seeing what checkstyle fails are there so you don't spam the commits on the pr.